### ISSUE TITLE
runv-containerd hangs

### ISSUE THREAD

**Issue by WeiZhang555** [hyperhq/runv#279]
Start runv-containerd and connect it with docker daemon, then execute: 

```
# cid=$(docker run -tid busybox sh) && docker exec -ti $cid echo hello 
```

docker daemon hangs. 

After some investigation, I found that at the time when it ran "docker exec", the container is actually not started/ready yet. 

I can throw an error out but that doesn't solve it. Could you fix it?

**Comment by wrouesnel** [comment]
I have a suspicion this might be related to my #377 as well.

**Comment by wrouesnel** [comment]
So I debugged this a bit tonight: it's definitely a runv side issue, and not a docker-sequencing issue.

If I add a delay to this line: https://github.com/hyperhq/runv/blob/master/supervisor/container.go#L209
by a simple `time.Sleep(time.Second*5)` then the problem goes away and the command works.

This is probably related to some other issues where the vm state management seems to get deadlocked if it's requested too quickly (and in turn, seems to affect notifications of container activities).

**Comment by wrouesnel** [comment]
```bash
$ cid=$(./docker run -tid busybox sh) && echo $cid && ./docker exec -ti $cid echo hello
1626908ea948b736eca9f6c565bd1a38492fbb409d630e13c1b96d7efb809a0f
hello
```

Solved by #379 now.

**Comment by laijs** [comment]
close it since #537 is merged.
